fg_lib/Kconfig

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NPM_FUEL_GAUGE
+	bool "Enable Nordic Fuel Gauge library"
+	depends on (!MINIMAL_LIBC)
```

Comment:
Could this be:
`select REQUIRES_FULL_LIBC`

Reply:
Yep, looks like this is a better alternative. Thanks!